base/sparse/sparsematrix.jl

```diff
@@ -795,7 +795,24 @@ end
 droptol!(A::SparseMatrixCSC, tol, trim::Bool = true) =
     fkeep!(A, (i, j, x) -> abs(x) > tol, trim)
 
+"""
+    dropzeros!(A::SparseMatrixCSC, trim::Bool = true)
```

Comment:
Good catch re. `SparseVector`s! I've updated the pull request with expanded tests and documentation for the `SparseVector` `dropzeros[!]` methods.